extensions/prototypes/labgraph_monitor/src/components/SettingPanel/GraphSettings.tsx

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { useState } from 'react';
+import { TabContext, TabList, TabPanel } from '@mui/lab';
+import {
+    Box,
+    Tab,
+    MenuItem,
+    InputLabel,
+    FormControl,
+    Button,
+    Stack,
+} from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import { MOCK } from '../../mocks';
+import { useDispatch, useSelector } from 'react-redux';
+import { RootState } from '../../redux/store';
+import { setMockGraph } from '../../redux/reducers/mock/mockReducer';
+import { setConnection } from '../../redux/reducers/ws/WSReducer';
+import WS_STATE from '../../redux/reducers/ws/enums/WS_STATE';
+
+const useStyles = makeStyles({
+    root: {
+        width: '100%',
+    },
+
+    tab: {
+        width: '50%',
+    },
+    tabLabel: {
+        fontSize: '.85rem',
+        fontWeight: 400,
+    },
+
+    textFieldLabel: {
+        fontSize: '.85rem',
+    },
+});
+
+/**
+ * A component that manages the settings of a graph.
+ *
+ * @returns {JSX.Element}
+ */
+const GraphSettings: React.FC = (): JSX.Element => {
+    const classes = useStyles();
+    const { connection } = useSelector((state: RootState) => state.ws);
+    const [value, setValue] = useState<string>('1');
+    const [mock, setMock] = useState<string>('');
+    const dispatch = useDispatch();
+
+    const handleChange = (_: React.SyntheticEvent, newValue: string) => {
```

Comment:
This seems a bit redundant, we could just have `(_, newValue) => setValue(newValue)` inside the `onChange`, rather than having a brand new function. Either remove it, or rename it to something less generic

Reply:
Done